t/02_request/14_uploads.t

```diff
@@ -182,5 +190,33 @@ for (qw/a b c/){
 }
 
 $resp =  dancer_response( 'POST', '/uploads', {files => $files});
-is length($resp->content), 3;
-like $resp->content, qr/a/;
+is $resp->content, 
+    join(';',"a:0cc175b9c0f1b6a831c399e269772661",
+    "b:92eb5ffee6ae2fec3ad71c777531578f",
+    "c:4a8a08f09d37b73795649038408b5f33"),
+    "Expected response for multi uploads";
+
+
+
+## test #23
+# create a file, 256MB of zeros
+$dest_file = File::Spec->catfile($dest_dir, "zeros");
+system("dd if=/dev/zero of=$dest_file count=250000 bs=1024");
```

Comment:
I agree that 256MB is a bit borderline, if this test is triggered by default. I don't see it as a problem if it's triggered by a dedicated ENV variable though.

DANCER_TEST_BIG_FILE_UPLOAD=1 prove -vl t/.....t  # something like that